rules/aws_cloudtrail_rules/aws_ipset_modified.py

```diff
@@ -0,0 +1,17 @@
+from panther_base_helpers import aws_rule_context
+
+IPSET_ACTIONS = ["CreateIPSet", "UpdateIPSet"]
+
+
+def rule(event):
+    if event.get("eventName", "") in IPSET_ACTIONS:
```

Comment:
we should generally couple `eventSource` and `eventName` with eventSource first which should lead to faster evaluation of the filter